library/Freckle/App/Database.hs

```diff
@@ -35,6 +35,13 @@ import Data.IORef
 import Data.Pool
 import qualified Data.Text as T
 import Database.Persist.Postgresql
+  ( SqlBackend
+  , SqlPersistT
+  , createPostgresqlPoolModified
+  , createSqlPool
+  , openSimpleConn
+  , runSqlPool
+  )
```

Comment:
Some versions of this module export a `Connection`, others don't. Explicit imports allow us to avoid the collision with `PostgreSQL.Simple` in the versions that do.